ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
     |  +--ro OTSiG-identifier    int16
     |  +--ro OTSiG-container
```

Comment:
I think we can remove this container. It is making the YANG tree depth longer but it does not help grouping attributes since all the attributes are within this container.

Reply:
Looking at the structures, it's correct that e.g. the container is not a presence contaiiner, but increase readability of the tree. It can be avoided in case it is confirmed the key identifier , representing an OTSIG with related list of OTSi.

Reply:
Hi Italo , let's do it as suggested .